packages/graphql/src/schema/constants.ts

```diff
@@ -23,3 +23,10 @@ export const DEPRECATE_NOT = {
         reason: "Negation filters will be deprecated in version 4.0.0, use the NOT operator to achieve the same behavior",
     },
 };
+
+export const DEPRECATE_INVALID_AGGREGATION_FILTERS = {
+    name: "deprecated",
+    args: {
+        reason: "Aggregation filters that are not relying on an aggregating function like 'LONGEST', 'SHORTEST', 'AVERAGE', 'SUM', 'MIN' and 'MAX' will be deprecated.",
```

Comment:
I think we don't need to give examples here, I would simplify!

```suggestion
        reason: "Aggregation filters that are not relying on an aggregating function will be deprecated.",
```

Reply:
Thank you, I was adding the Changeset but you were too fast! 😆